docs/csa/audit-and-compliance.md

```diff
@@ -0,0 +1,170 @@
+---
+id: audit-and-compliance
+title: Audit and Compliance
+sidebar_label: Audit and Compliance
+description: Learn how to use Cloud Security Analytics to assist with audit and compliance. 
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+To use Cloud Security Analytics for audit and compliance, you can use [pre-built apps](#pre-built-apps-for-audit-and-compliance), or [build your own queries](#build-queries-for-audit-and-compliance).
+
+## About audit and compliance
+
+Compliance standards are enforced by various governments and organizations to protect end users and keep information safe. Meeting compliance standards is usually the first step in making sure your apps and data are secure. However, in most cases additional security measures, such as [threat monitoring](/docs/csa/threat-detection-and-investigation/), are also recommended.
+
+A compliance audit is an inspection of an organization to check whether compliance standards are being met. There are two types of audits Sumo Logic Security Analytics can support:
+* **Internal audit**. Conducted by members of your own organization, as a practice to make sure that you’ll pass the inspection. 
+* **External audit**. Where a government or other independent authority checks your data to make sure you’re meeting compliance standards. Both internal and external audits can be scheduled or they can be random.
+
+Both internal and external audits can be scheduled or they can be random. For example, you can use Sumo Logic's queries, alerts, and dashboards to monitor your data, and make sure only authorized users have access to sensitive information.
+
+Because Sumo Logic itself meets many security compliance standards, including HIPAA, FedRAMP-Moderate Authorized, ISO 27001, and many others, you can feel safe ingesting your data into the Sumo Logic platform.
```

Comment:
```suggestion
Because Sumo Logic itself meets many security compliance standards, including HIPAA, FedRAMP-Moderate Authorized, ISO 27001, and many others, you can feel safe ingesting your data into our platform.
```